src/index.html

```diff
@@ -10,7 +10,57 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    data-qa="big"
+    class="form"
+    >
+      <label
+        for="big__search"
+        class="search__label"
+      >
+        <img
+          src="images/Search.svg"
+          alt="search icon"
+          class="big__icon"
+        >
+      </label>
+      <input
+        class="big__search"
+        type="search"
+        name="bigSearch"
+        id="big__search"
```

Comment:
Change the 'id' name to `big-search` or `bigSearch` to point right label `for` tag name.